app-conf/elephant.conf

```diff
@@ -36,4 +36,20 @@ jvm_args="-Devolutionplugin=enabled -DapplyEvolutions.default=true"
 #   /ping
 #   /healthcheck
 #   /metrics
-metrics=true
\ No newline at end of file
+metrics=true
+
+#
+# Property enables an agent jar to be loaded along with the Dr. Elephant application.
+# The intention with this property is to have the agent publish metrics to other
+# applications, although it can do anything else. This property is disabled by default
+# and users wishing to make use of it should provide their own implementation of the agent.
+#
+# More information on writing a jvm agent can be found under the following.
+# https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/package-summary.html
+#
+# Sample configuration of this property is shown below.
+# metrics_agent_jar="-javaagent:lib/your_agent.jar"
+#
+# Sample configuration of the agent with additional options.
+# metrics_agent_jar="-javaagent:lib/your_agent.jar=app-name=dr-elephant,app-host=foo"
```

Comment:
Yes, we can do that. Separate property is defined to provide more info to the user as to what it does and how to use it.